source/common/network/address_impl.cc

```diff
@@ -47,6 +47,10 @@ std::string friendlyNameFromAbstractPath(absl::string_view path) {
 
 } // namespace
 
+static inline const SocketInterface* sockInterfaceOrDefault(const SocketInterface* sock_interface) {
```

Comment:
style nit: put this in unnamed namespace above, remove `static`, doesn't have to be `inline`?

Reply:
It's a C reflex of mine that I use instead of macros. Goal would be to convince the compiler to avoid extra function calls when not needed. But obviously, no strong opinion here ...

Reply:
C++ `inline` isn't guaranteed to be inlined, all are decided by compiler, it's more of working around one-definition rule.

Reply:
Understood!